evap/staff/views.py

```diff
@@ -914,6 +914,35 @@ def user_delete(request, user_id):
 
 
 @staff_required
+def user_merge_selection(request):
+    form = UserMergeSelectionForm(request.POST or None, request.FILES or None)
+
+    if form.is_valid():
+        main_user = form.cleaned_data['main_user']
+        other_user = form.cleaned_data['other_user']
+        return redirect('staff:user_merge', main_user.id, other_user.id)
+    else:
+        return render(request, "staff_user_merge_selection.html", dict(form=form))
```

Comment:
yes, i thought about that. it saves entering one username. but it would preferably have both options you mentioned and i don't know whether it's worth it. i will find out when actually using it on production and add it later if necessary :)